k8s/demo/common/rpe/draft-store-service.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: draft-store-service-demo
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service-demo
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: draft-store-service
+    version: 1.6.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/draft-store/service:prod-e0865856
+      environment:
+        IDAM_URL: https://idam-api.demo.platform.hmcts.net
```

Comment:
Not needed as we have : https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L12
```suggestion
```